cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala

```diff
@@ -72,7 +81,7 @@ object CqlParser extends JavaTokenParsers
 
     def integer = wholeNumber
 
-    def float = floatingPointNumber
+    def float = """[+-]?[0-9]*((\.[0-9]+([eE][+-]?[0-9]+)?[fF]?)|([fF])|([eE][+\u200C\u200B-]?[0-9]+))\b""".r
```

Comment:
The float regex specified by Datastax docs is `-?[0-9]+(\.[0-9]*)?([eE][+-]?[0-9+])?`, can we use it instead?